includes/definitions/ge-digitalenergy.yaml

```diff
@@ -0,0 +1,16 @@
+os: ge-digitalenergy
```

Comment:
tbh I don't even think there's an OS behind it. It's a simple extension card in the UPS to provide basic SNMP support. it *could* be changed to ge-ups